back/src/controllers/messageController.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable quotes */
+import { messageService } from "../services/messageService.js";
+
+async function createMessage(req, res, next) {
+  const { name } = req.body;
+  const room = await roomService.createRoom(name);
+
+  res.send(room);
+}
```

Comment:
it seems like u r not using `createMessage` from this controller anywhere

Reply:
Yes, sorry, it was undeleted copy/paste)) This controller function is unnecessary because of saving messages through the WebSocket connection.
Fixed.